File: functions/src/domain/projectInvitation.bak/repositories/factory/ProjectInvitationRepositoryFactoryContext.ts
/**
 * ProjectInvitationRepository Factory 컨텍스트
 */
export interface ProjectInvitationRepositoryFactoryContext {
  /**
   * 사용자 ID (인증된 사용자)
   */
  userId: string;

  /**
   * 프로젝트 ID (선택사항)
   */
  projectId?: string;
}

File: domain/src/main/java/com/example/domain/repository/factory/context/ProjectInvitationRepositoryFactoryContext.kt
package com.example.domain.repository.factory.context

import com.example.domain.model.vo.CollectionPath

/**
 * ProjectInvitation Repository Factory Context
 * DDD pattern에 따라 collection path를 관리합니다.
 */
class ProjectInvitationRepositoryFactoryContext(
    override val collectionPath: CollectionPath = CollectionPath.projectInvitations()
) : DefaultRepositoryFactoryContext